src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -763,9 +763,10 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode)
                     ->fixXmlConfig('path')
                     ->children()
                         ->arrayNode('fallbacks')
+                            ->info('Defaults to the value of "default_locale".')
                             ->beforeNormalization()->ifString()->then(function ($v) { return [$v]; })->end()
                             ->prototype('scalar')->end()
-                            ->defaultValue(['en'])
+                            ->defaultValue([])
```

Comment:
Isn't this a BC break?

Reply:
Not if default_local was set to the same value, right? Should always be the case?

Reply:
I don't think so. For a German site I would set `default_locale` to `de` but keep the fallback set to `en` if my team writes English messages and translations are provided later by a separate team.

Reply:
It's an edge case that can be fixed very easily, and deserves to be explicit IMHO. The current behavior create a WTF situation where you change the default_locale, but it's not taken into account by Symfony itself without having to change another setting.